flow/benchmarks/figureeight2.py

```diff
@@ -63,7 +63,6 @@
     # network-related parameters (see flow.core.params.NetParams and the
     # scenario's documentation or ADDITIONAL_NET_PARAMS component)
     net=NetParams(
-        no_internal_links=False,
```

Comment:
Okay, my one comment is, if something is the default for every single file, it should actually just be set as a default in the Class definition. This would have made it a lot easier to remove all of these.

Reply:
agreed